kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml

```diff
@@ -1058,6 +1058,37 @@ paths:
 
 components:
   schemas:
+    ErrorResponse:
+      description: Error object that will be returned with 4XX and 5XX HTTP statuses
+      type: object
+      properties:
+        code:
+          type: integer
+          description: Internal error code (can be used for message formatting & localization on UI)
+        message:
+          type: string
+          description: Error message
+        timestamp:
+          type: number
+          description: Response unix timestamp in ms
+        requestId:
+          type: string
+          description: Unique server-defined request id for convenient debugging
+        fieldErrors:
+          type: array
+          items:
+            $ref: '#/components/schemas/FieldError'
```

Comment:
`fieldErrors:` might be '#/components/schemas/FieldError' without any extra array wrappers
```
fieldErrors:
  $ref: '#/components/schemas/ContactInfo'
```

Reply:
@workshur we can have several fields validation errors, so this is why array is used 
example
```
  "fieldErrors": [
    {
      "fieldName": "name",
      "restriction": "must not be null"
    },
    {
      "fieldName": "partitions",
      "restriction": "should be > 0"
    },
  ]
```

Reply:
In this case, restriction filed should contain an array of messages. It is not critical for me.

Reply:
@workshur renamed fieldErrors to field**s**Errors to highlight that errors for several fields can be returned